<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>130</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Poistettava j√§senyys</string>
   </property>
  </widget>
  <widget class="QComboBox" name="removeMembershipComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="removeMembershipCancelPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Peruuta</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="removeMembershipPushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Poista</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
